services/storehost/extMgr.go

```diff
@@ -105,6 +134,9 @@ type (
 		//    ref is 0, the 'extentLock' needs to be acquired and the ref re-checked
 		//    before initiating clean-up/tear-down
 
+		// count of close/cleanup callbacks that have reference on the structure
+		cleanupRef uint32
```

Comment:
hmm.. don't really like this another refCounting for callbacks.. lets talk about this today.